app/ApiObjects/Comment.php

```diff
@@ -0,0 +1,39 @@
+<?php namespace App\ApiObjects;
+
+use App\ApiObjects\User as ApiUser;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Contracts\Support\Jsonable;
+use Illuminate\Support\Facades\Auth;
+
+class Comment implements Arrayable, Jsonable
+{
+    protected $comment;
+
+    public function __construct($comment)
+    {
+        $this->comment = $comment;
+    }
+
+    public function toJson($options = 0)
```

Comment:
Looks like `$options` parameter isn't used, so we can drop it. I have no idea why I had that param in the `Episode`.